omod/src/main/webapp/resources/htmlforms/PatientReferral.xml

```diff
@@ -182,6 +222,20 @@
                 });
             });
 
+            function getPatientTransferEncounterTypes(encounterType) {
```

Comment:
@dbaluku it is not clear what this method does, are you getting the encouterTypes or the encounters of a specific encounterType? 

Maybe add a description of what the method does

Reply:
Done